.changeset/weak-knives-build.md

```diff
@@ -0,0 +1,19 @@
+---
+"@salt-ds/core": patch
```

Comment:
There is some new feature, it would be minor.

However, given the uncertainty of the design choice, this may need to be lab first?